README.md

```diff
@@ -19,9 +19,9 @@ releases](https://github.com/johnno1962/InjectionIII/releases) of the app
 or from the [Mac App Store](https://itunes.apple.com/app/injectioniii/id1380446739?mt=12) 
 and adding the code below somewhere in your app to be executed on
 startup (it is no longer necessary to actually run the app itself). It's also
-important to add "-Xlinker -interposable" (without the double quotes and on separate
-lines) to the "Other Linker Flags" of all targets in your project (for the `Debug`
-configuration only) to enable "interposing" (see the explanation below).
+important to add the options `-Xlinker` and `-interposable` (without double quotes 
+and on separate lines) to the "Other Linker Flags" of all targets in your project 
+(for the `Debug` configuration only) to enable "interposing" (see the explanation below).
```

Comment:
How about adding a screenshot:

<img width="451" alt="image" src="https://github.com/johnno1962/InjectionIII/assets/341895/4b422e93-2ed9-4db2-b128-11043ea0cdc5">

and/or detailing the final result on the `.pbxproj` file?

```
OTHER_LDFLAGS = (
  "-Xlinker",
  "-interposable",
);
```

Reply:
You're quite right, This is important. I've added this image to the "codesigning" branch.